db/seed.js

```diff
@@ -0,0 +1,61 @@
+export const addTables = db => {
+    db.transaction( tx => {
+        tx.executeSql(addMonstersTable);
+        tx.executeSql(addSessionsTable);
+        tx.executeSql(addActiveMonstersTable);
+    },transactionError,transactionSuccess)
+};
+
+
+export const seedMonstersTable = db => {
+    db.transaction( tx => {
+        addTestMonster(tx,'Harold');
+    },transactionError,transactionSuccess)
+};
+
+const transactionError = (err) => console.log("ERROR",err);
+const transactionSuccess = () => console.log("SUCCESS");
+
+
+const addMonstersTable = `
+    CREATE TABLE IF NOT EXISTS monsters (
+        id INTEGER PRIMARY KEY,
+        name VARCHAR(20),
+        health INTEGER Default 0,

```

Comment:
Potentially might be better to set these defaults to 10 since that's the base 0 stat, much more likely to stay at 10. If things are set to 0, then everything will 100% have to be changed. This is just a QoL thing though

Reply:
Okay. What will be the integer range of the stats?

Reply:
Stats can be somewhere between 0 and say... 50? Almost always stats are between 0 and 20 but I don't see a reason to limit it unless it'll cause memory issues